dashboard/app/models/script.rb

```diff
@@ -12,7 +12,7 @@
 #  properties      :text(65535)
 #  new_name        :string(255)
 #  family_name     :string(255)
-#  published_state :string(255)      default("in_development"), not null
+#  published_state :string(255)      default("in_development")
```

Comment:
just checking my understanding that this means (1) scripts created via /s/new will still have a default published state of "in_development", and (2) the published_state can be set to null explicitly, e.g. `Script.find_by_name(...).update!(published_state: nil)`? apologies, my local DB is in a bad state right now so I cannot verify this myself at the moment.